caom2utils/caom2utils/tests/test_obs_blueprint.py

```diff
@@ -92,7 +92,7 @@ def test_obs_blueprint():
                 position_axes=(1, 2), energy_axis=3, polarization_axis=4, time_axis=5, obs_axis=6, custom_axis=7
             )
         ).count('\n')
-        == 90
+        == 89  # TODO why?
```

Comment:
TODO why - Because `Plane.productID = 'RUNID'` was removed?

Reply:
Yes. Makes sense. I forgot to revisit it. Thanks!